context/app/static/js/components/Detail/Protocol/Protocol.jsx

```diff
@@ -3,21 +3,22 @@ import PropTypes from 'prop-types';
 import Divider from '@material-ui/core/Divider';
 
 import { LightBlueLink } from 'shared-styles/Links';
+import useProtocolData from 'hooks/useProtocolData';
 import { StyledPaper } from './style';
 import SectionHeader from '../SectionHeader';
 import SectionItem from '../SectionItem';
 import SectionContainer from '../SectionContainer';
 
 function ProtocolLink(props) {
-  const { protocolUrl: urlMinusProtocol } = props;
+  const { doi } = props;
   return (
     <SectionItem label="Protocol URL">
-      {urlMinusProtocol ? (
-        <LightBlueLink href={`https://${encodeURI(urlMinusProtocol)}`} target="_blank" rel="noopener noreferrer">
-          {urlMinusProtocol}
+      {doi ? (
+        <LightBlueLink href={`https://${encodeURI(doi)}`} target="_blank" rel="noopener noreferrer">
```

Comment:
If this works, then `doi` isn't just a a DOI, it's resolver_hostname + doi. 

https://en.wikipedia.org/wiki/Digital_object_identifier
> `prefix/suffix`
> ... The prefix usually takes the form `10.NNNN` ...